packages/core-data/src/actions.js

```diff
@@ -974,6 +985,7 @@ export function receiveDefaultTemplateId( query, templateId ) {
 export const receiveRevisions =
 	( kind, name, recordKey, records, query, invalidateCache = false, meta ) =>
 	async ( { dispatch, resolveSelect } ) => {
+		logEntityDeprecation( kind, name, 'receiveRevisions' );
```

Comment:
The `attachment` post type doesn't support revisions by default, but I think it's a good safeguard for future deprecations.